README.md

```diff
@@ -42,8 +42,6 @@ notaryproject.dev is licensed under an [Apache 2.0 license](./LICENSE).
 
 The Notary Project documentation (e.g., `.md` files in the `/content/docs` folder) is licensed under a [CC-BY-4.0 license](./LICENSE).
 
-The notaryproject.dev site uses [Docsy](https://www.docsy.dev/), and was cloned from the [etcd-io/website](https://github.com/etcd-io/website/) repo to capture CNCF specific edits.
```

Comment:
We already mention Docsy elsewhere in this page, and I don't feel that the rest of the sentence is relevant info to NP website contributors.